src/components/ICA/ICA.js

```diff
@@ -65,11 +69,29 @@ const ICA = ({
   label,
   locale,
   percentage,
+  size,
+  trending,
   truncate,
   total,
   value,
   ...other
 }) => {
+  const isSize = sizeValue => size === sizeValue;
+  const isLarge = isSize('large') || isSize('lg');
+  const isXLarge = isSize('xlarge') || isSize('xlg');
+
+  let renderIcon = ArrowUp16;
+  if (isLarge) {
+    renderIcon = ArrowUp20;
+  } else if (isXLarge) {
+    renderIcon = ArrowUp24;
+  }
+
+  const ICAClasses = classnames(className, {
+    [`${namespace}--lg`]: isLarge,
+    [`${namespace}--xlg`]: isXLarge,
```

Comment:
For naming consistency with [Carbon's sizes](https://react.carbondesignsystem.com/?path=/docs/components-button--default#button-size):

```suggestion
    [`${namespace}--xl`]: isXLarge,
```